code/src/Turtles.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Turtles.css'
+import img from './images/ninja.png'
+
+const Turtle = props => {
+    const {setTurtles, turtle} = props
+    return (
+        <div className="turtles-wrapper">
+            <h2 tabIndex="0">and, finally best ninja?</h2>
+            <img
+                tabIndex="0"
+                className="turtles-image"
+                alt="visualization of teenage mutant ninja turtles"
+                src={img}
+            />
+            <label htmlFor="ninja">Choose your TMNT from the select menu!</label>
```

Comment:
Same comment here as on the other questions about screenreader not reading the label.